src/Assets/styleController.ts

```diff
@@ -6,6 +6,7 @@ export interface ITheme {
   terraCottaColor: string;
   midBlueColor: string;
   secondaryBackgroundColor: string;
+  lightLavenderBackground: string;
```

Comment:
Can you use `--secondary-background-color` from [Marianna/734/typography-updates](https://github.com/Gary-Community-Ventures/benefits-calculator/tree/Marianna/734/typography-updates)